docs: add pipedrive documentation (#2743)
apify/apify-docs: sources/platform/integrations/workflows-and-notifications/pipedrive.md

- **Organization detail**: the Apify enrichment panel shows the latest enrichment status and lets you create or run an enrichment for that organization.
- **Settings page**: manage your Apify connection and browse the **Description**, **Interactive features**, **Setup and installation**, and **Resources** tabs.

![The Apify enrichment entrypoint in the organization list three-dots menu](../images/pipedrive/organization-list-entrypoint.webp)

![The Apify enrichment panel in the Pipedrive organization detail view](../images/pipedrive/organization-detail-panel.webp)

## Create an enrichment configuration

An enrichment configuration defines the Actor settings, including the number of contacts to find per organization, and how Actor output maps to your Pipedrive fields. You build one through a guided wizard. A configuration is not tied to specific organizations, so you can reuse it with any organization.

### Step 1: Set up the enrichment

Configure what the enrichment collects:

- **Leads per organization**: how many contacts to find per organization, from 1 to 100.
- **Departments**: which company departments to target.
- **Social media profiles**: which profiles to collect.

> - **Deduplication mode**: how existing Pipedrive values are handled. You have to choose one of:

Reviewer: Two small style tweaks: we favor active voice, so "are handled" becomes "the app handles"; and "You have to choose one of" is more direct as just "Choose one of".

```suggestion
- **Deduplication mode**: how the app handles existing Pipedrive values. Choose one of:
```

    - **Overwrite existing data**
    - **Auto-fill only empty fields**

>     - **Skip existing records**

Reviewer: **This fails CI (`pnpm lint:md`, rule `MD007/ul-indent`).** Nested bullets should be indented 2 spaces, not 4. Dropping the indent to 2 fixes it.

```suggestion
  - **Overwrite existing data**
  - **Auto-fill only empty fields**
  - **Skip existing records**
```

- **Automatically enrich new organizations**: when enabled, the app automatically enriches newly added organizations using this configuration.

For more details you can read the [Contact Details Scraper](https://apify.com/vdrmota/contact-info-scraper) Actor documentation.

![The Actor configuration step of the enrichment wizard](../images/pipedrive/wizard-configuration.webp)

### Step 2: Map fields

Map the fields from the Actor output to your Pipedrive fields. Each mapping row pairs an Apify field with a Pipedrive field and its data type.

You need at least one fully configured mapping row to continue. The wizard discards any row that isn't fully filled.

![The field mapping step of the enrichment wizard showing Apify fields mapped to Pipedrive fields](../images/pipedrive/wizard-field-mapping.webp)

### Step 3: Review and save

Review a summary of the configuration, then save it and optionally run the enrichment right away.

:::caution Organizations need a website

> Contact enrichment starts from an organization's website URL. Organizations without a website are skipped, and the review step lists how many will be skipped.

Reviewer: Active voice reads cleaner here — make "the app" the subject instead of "are skipped".

```suggestion
Contact enrichment starts from an organization's website URL. The app skips organizations without a website, and the review step lists how many it will skip.
```

:::

![The review step of the enrichment wizard summarizing the configuration before saving](../images/pipedrive/wizard-review.webp)

## Run enrichments

You can run a saved configuration in two ways:

- **On demand**: select the organizations you want to enrich in the organization list, then select **Run now** on a configuration and confirm the organization selection before running. You can also select **Create enrichment** from the Apify panel in an organization's detail view to enrich that organization.
- **Automatically**: enable **Automatically enrich new organizations** in a configuration. The app then runs enrichments using that configuration whenever a new organization is added to your Pipedrive account.

Enrichment runs are asynchronous. The run starts in the background, and the results appear in Pipedrive once it finishes.

## View enrichment results and history

Each run reports its status and a summary of what changed:

- **Statuses**: a run shows as **Processing** while it works, then **Done** or **Failed**.

> - **Results summary**: items (contacts + organizations) added, updated, skipped, and failed.

Reviewer: A `+` sign looks informal in prose — spell it out as "and".

```suggestion
- **Results summary**: items (contacts and organizations) added, updated, skipped, and failed.
```

You can review past runs in the enrichment history from the organization list modal, and open a single run to see its configuration, results, and enrichment data.

![The enrichment settings and history modal showing past runs with their status and results summary](../images/pipedrive/organization-list-modal.webp)

## Limitations

> - Organizations without a website URL are skipped, because contact enrichment starts from the organization's website.

Reviewer: Active voice: let "the app" do the skipping.

```suggestion
- The app skips organizations without a website URL, because contact enrichment starts from the organization's website.
```

- The number of leads per organization must be between 1 and 100.
- A configuration needs at least one fully configured field mapping before you can save it.

> - Enrichment runs asynchronously, so results are written back to Pipedrive only after the run finishes.

Reviewer: Active voice: "the app writes" instead of "results are written back".

```suggestion
- Enrichment runs asynchronously, so the app writes results back to Pipedrive only after the run finishes.
```
